app/models/period.rb

```diff
@@ -185,6 +185,16 @@ def to_s(format = :mon_year)
     end
   end
 
+  def to_dhis2
+    # DHIS2 has its own period string formats
+    # https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/introduction.html#webapi_date_perid_format
+    if quarter?
+      "#{value.year}Q#{value.number}"
```

Comment:
I think we could add this to time and date formats as well. I think it would just be `"%YQ%m"`.